AndroidAnnotations/androidannotations-core/androidannotations/src/test/java/android/support/v7/preference/PreferenceFragmentCompat.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (C) 2010-2015 eBusiness Information, Excilys Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed To in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package android.support.v7.preference;
+
+public class PreferenceFragmentCompat extends android.app.Fragment {
+
+	public void addPreferencesFromResource(int preferencesResId) {
+
+	}
+
+	public android.support.v7.preference.Preference findPreference(java.lang.CharSequence key) {
```

Comment:
You are right. I quickly copied this from the decompiled source, that's why FQCNs are here.